app/serializers/incoming_email_details_serializer.rb

```diff
@@ -17,7 +17,7 @@ def initialize(incoming_email, opts)
 
   def error
     if @error_string.blank?
-      "Unrecognized Error"
+      I18n.t("emails.incoming.unrecognized_error")
```

Comment:
Neat, I was wanting to do something in that form but couldn't work it out... now I know!